### ISSUE TITLE
Scaffolding doesn't include tinyint(3) unsigned column in the map

### ISSUE THREAD

**Issue by AccessViolator** [PomeloFoundation/Pomelo.EntityFrameworkCore.MySql#655]
### Steps to reproduce
Have a table with `tinyint(3) unsigned` column, run scaffolding.

### The issue
The entity is generated correctly and includes property of type byte, however the property is missing from the entity's map in DbContext.OnModelCreating()

### Further technical details

MySQL version: 8.0.11
Operating system:  Win 7 x64
Pomelo.EntityFrameworkCore.MySql version: 2.1.1

**Comment by mguinness** [comment]
Byte maps to tinyint unsigned.  Were you expecting HasColumnType with display width?

**Comment by AccessViolator** [comment]
I was expecting .Property() entry with or without HasColumnType().
Are you saying it's not there because `tinyint(3) unsigned` is the default mapping for a byte?

**Comment by mguinness** [comment]
It appears that this is caused by the fix for https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql/issues/394 which strips out the display width for unsigned integers.

**Comment by lauxjpn** [comment]
This was very likely fixed by #896. I will check this later at some point.

**Comment by lauxjpn** [comment]
I can confirm that this is fixed now in `3.0.0`.

I ran the following SQL:

```sql
CREATE TABLE `Issue655`.`IceCream` (
  `Id` INT NOT NULL AUTO_INCREMENT,
  `WeightInGrams` TINYINT(3) UNSIGNED NOT NULL,
  PRIMARY KEY (`Id`)
);
```
Then i scaffolded the database:
```c#
dotnet ef dbcontext scaffold "server=127.0.0.1;user id=root;password=;port=3306;database=Issue655" Pomelo.EntityFrameworkCore.MySql -c Context
```
The following classes were generated:
```c#
public partial class IceCream
{
    public int Id { get; set; }
    public byte WeightInGrams { get; set; } // <-- correct CLR type
}

public partial class Context : DbContext
{
    public Context()
    {
    }

    public Context(DbContextOptions<Context> options)
        : base(options)
    {
    }

    public virtual DbSet<IceCream> IceCream { get; set; }

    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
    {
        if (!optionsBuilder.IsConfigured)
        {
            optionsBuilder.UseMySql(
"server=127.0.0.1;user id=root;password=;port=3306;database=Issue655");
        }
    }

    protected override void OnModelCreating(ModelBuilder modelBuilder)
    {
        modelBuilder.Entity<IceCream>(entity =>
        {
            entity.Property(e => e.Id)
                .HasColumnType("int(11)");
            entity.Property(e => e.WeightInGrams)
                .HasColumnType("tinyint(3) unsigned"); // <-- correct MySQL type
        });

        OnModelCreatingPartial(modelBuilder);
    }

    partial void OnModelCreatingPartial(ModelBuilder modelBuilder);
}
```
Finally, the following test executed successfully:
```c#
internal class Program
{
    private static void Main()
    {
        using var context = new Context();

        context.Database.EnsureDeleted();
        context.Database.EnsureCreated();

        context.IceCream.Add(new IceCream { WeightInGrams = 68 });
        context.SaveChanges();

        var iceCream = context.IceCream.Single();

        Debug.Assert(iceCream.WeightInGrams is byte);
        Debug.Assert(iceCream.WeightInGrams == 68);
    }
}
```
